src/d_deh.c

```diff
@@ -828,6 +874,48 @@ deh_strs deh_strlookup[] = {
   {&startup4,"STARTUP4"},
   {&startup5,"STARTUP5"},
   {&savegamename,"SAVEGAMENAME"},  // Ty 05/03/98
+
+// [FG] obituaries
+
+  {&s_OB_CRUSH, "OB_CRUSH"},
```

Comment:
Do other ports use these strings or is it just for woof?

Reply:
Nope the strings are taken from gzdoom:
https://github.com/ZDoom/gzdoom/blob/22271d146a0bca6b432f458c96c4ca18a1b6b143/wadsrc/static/language.csv#L1191

Reply:
Also, this feature is at least in Eternity and Doom Retro.